packages/lit-starter-js/docs-src/api.11ty.cjs

```diff
@@ -2,89 +2,131 @@
  * This page generates its content from the custom-element.json file as read by
  * the _data/api.11tydata.js script.
  */
-module.exports = class Docs {
-  data() {
-    return {
-      layout: 'page.11ty.cjs',
-      title: '<my-element> ⌲ Docs',
-    };
-  }
+ module.exports = class Docs {
+   data() {
+     return {
+       layout: 'page.11ty.cjs',
+       title: '<my-element> ⌲ Docs',
+     };
+   }
 
-  render(data) {
-    const customElements = data.api['11tydata'].customElements;
-    const tags = customElements.tags;
-    return `
+   render(data) {
+     const manifest = data.api['11tydata'].customElements;
+     const elements = manifest.modules.reduce(
+       (els, module) =>
+         els.concat(
+           module.declarations?.filter((dec) => dec.customElement) ?? []
```

Comment:
So the most correct thing to do here is to get the exports, filter for custom element defines, then resolve those to their exports... but in the starter kit we know we're exporting all the declarations, so it'll have the same effect. We might just want to do the more proper thing later to account for projects that grow from this.

Reply:
Yeah let's tackle that later.